tcpdf.php

```diff
@@ -21201,7 +21332,9 @@ public function deletePage($page) {
 			return false;
 		}
 		// delete current page
-		unset($this->pages[$page]);
+		if (isset($this->pages[$page])) {
+			unset($this->pages[$page]);
```

Comment:
A bit too much indentation
Same below

Reply:
Sorry, I don't understand this comment.  It seems to match the other if statements.